pipeline/data_merging/brca_pseudonym_generator.py

```diff
@@ -75,7 +75,10 @@ def from_field_or_compute(row):
         computed = compute_hgvs(row)
 
         if not computed:
-            return cdna_from_cdna_field(row)
+            cDNA = cdna_from_cdna_field(row)
+            if cDNA is None:
+                logging.info("No calculable cDNA for row: " + str(row))
```

Comment:
since we moved to python3, you can also use string interpolation like this `f"No calculable cDNA for row: {row}"`